docs/apis/streaming/connectors/kinesis.md

```diff
@@ -299,3 +299,41 @@ Otherwise, the returned stream name is used.
 Other optional configuration keys can be found in `KinesisConfigConstants`.
 		
 		
+### Using non-AWS Kinesis Endpoints
+
+It is sometimes desirable to have Flink operate as a consumer or producer against a non-AWS Kinesis endpoint such as kinesalite; this is especially useful when performing functional testing of a Flink application. The AWS endpoint that would normally be inferred by the AWS region set in the Flink configuration must be overriden via a configuration property.
```

Comment:
Yes, thanks, I wasn't sure what the policy is for referencing other non-Apache projects.